File: youngen/week7/1920.js
const fs = require('fs');
const input = fs.readFileSync("youngen/input.txt").toString().trim().split("\n");
const [n, stringN, m, stringM] = input
const arrN = stringN.split(' ')
const arrM = stringM.split(' ')
arrM.forEach((m)=>{
  if(arrN.includes(m)){
    console.log(1)
  } else {console.log(0)}
})

File: nagyum/week7/1920.py
import sys
input=sys.stdin.readline

n=int(input())
a=set(map(int, input().split())) #list로 하니까 시간초과가 뜸 왜지
m=int(input())
b=list(map(int,input().split()))

for i in b:
    if i in a:
        print(1)
    else:
        print(0)